### PR TITLE
Added Buffer fees

### PR DESCRIPTION
**Pull request by bufferfinance**

### CODE REVIEW

**Comment by llamatester** [comment]
The buffer adapter exports: 
        
 
 _______________________________________
    ------ ERROR ------
    Error: Cannot find module '/home/runner/work/dimension-adapters/dimension-adapters/fees/buffer'
    Require stack:
    - /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts
        at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
        at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
        at Function.Module._load (node:internal/modules/cjs/loader:841:27)
        at Module.require (node:internal/modules/cjs/loader:1067:19)
        at require (node:internal/modules/cjs/helpers:103:18)
        at /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:31:51
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:31:28 {
      code: 'MODULE_NOT_FOUND',
      requireStack: [
        '/home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts'
      ]
    }
    Done in 2.96s.

**Comment by bufferfinance** [comment]
home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts - No changes to this file have been made.Any suggestions as to how i can fix this and pass the check?

**Comment by dtmkeng** [comment]
@bufferfinance  hi seem folder space should trailing should remove space folder 
image

**Comment by llamatester** [comment]
The buffer adapter exports: 
        
 
 _______________________________________
    Fees for 7/2/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 17/10/1971
    NO METHODOLOGY SPECIFIED
    Timestamp: 1675814398
    Daily fees: 3525.294659
    Daily revenue: 29219.86
    
    
    
    
    Done in 4.18s.

**Comment by dtmkeng** [comment]
@bufferfinance  could you check why daily fee more than daily rev ?

**Comment by bufferfinance** [comment]
Because a portion of the fees is added to the public liquidity pool, so the revenue is just a part of the fees

**Comment by bufferfinance** [comment]
The daily fee is lesser than the daily revenue in the above screenshot posted. Can you re-check the values?

**Comment by dtmkeng** [comment]
> The daily fee is lesser than the daily revenue in the above screenshot posted. Can you re-check the values?

oh sorry I mean daily fee less than daily rev,

**Comment by bufferfinance** [comment]
Yes, the revenue includes amount that is added to the pool too. The fees is a part of the revenue

**Comment by dtmkeng** [comment]
> Yes, the revenue includes amount that is added to the pool too. The fees is a part of the revenue

you mean, Isn't the rev all from fee ?

**Comment by bufferfinance** [comment]
Revenue is the net income while the fees is the profit of the platform

**Comment by 0xngmi** [comment]
fees = all money that users pay to the protocol (this includes money paid to LPs)
revenue = money that the protocol keeps (and usually goes to treasury or to gov token holders)

so yes, usually fees > revenue, as the protocol cant make more money than what the users are paying

**Comment by llamatester** [comment]
The buffer adapter exports: 
        
 
 _______________________________________
    Fees for 8/2/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 17/10/1971
    NO METHODOLOGY SPECIFIED
    Timestamp: 1675900798
    Daily fees: 23097.91
    Daily revenue: 2659.00425
    
    
    
    
    Done in 5.09s.

**Comment by bufferfinance** [comment]
Let us know if there is any other correction to make in order to merge it

File: fees/buffer/index.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    };
+  };
+};
+
+
+const adapter: Adapter = {
+  adapter: {
+    [CHAIN.ARBITRUM]: {
+        fetch: graphs(endpoints)(CHAIN.ARBITRUM),
+        start: async ()  => 56585555 ,
```
Comment (dtmkeng) [inline fees/buffer/index.ts]: could you change start `56585555` to unix timestamp
if use this start time is `Sun Oct 17 1971 22:12:35 GMT+0000`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: fees/buffer/index.ts (hunk 1 above)
Comment (bufferfinance) [reply fees/buffer/index.ts]: This is done

**Comment by llamatester** [comment]
The buffer adapter exports: 
        
 
 _______________________________________
    Fees for 8/2/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 29/1/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1675900798
    Daily fees: 52390.31
    Daily revenue: 7173.630294
    
    
    
    
    Done in 3.98s.

File: fees/buffer/index.ts (hunk 1 above)
Comment (bufferfinance) [reply fees/buffer/index.ts]: Let us know if anything else

**Pull request merged**